openapi/components/paths.yaml

```diff
@@ -12,6 +12,8 @@
   $ref: "./paths/authentication.yaml#/paths/~1auth~1twofactorauth~1totp~1verify"
 "/auth/twofactorauth/otp/verify":
   $ref: "./paths/authentication.yaml#/paths/~1auth~1twofactorauth~1otp~1verify"
+"/auth/twofactorauth/emailotp/verify":
+  $ref: "./paths/authentication.yaml#/paths/~1auth~1twofactorauth~emailotp~1verify"
```

Comment:
`~` should be `~1`.

This is because forwardslashes cannot be used in this place, as they indicate folders, while the slash is inside the name of a variable inside the file, so OpenAPI (and possibly YAML in general) uses ~1 as replacement.